lib/require-patch.js

```diff
@@ -2,7 +2,9 @@
 
 const Module = require('module')
 const path = require('path')
-const ao = require('./index')
+
+const { loggers } = require('./loggers')
```

Comment:
Curious what the curly braces do?

Reply:
Object [deconstruction syntax](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment)
```
const o = { foo: 'bar' }
const { foo } = o
console.log(foo) // output: bar
```
of what was [exported from `loggers`](https://github.com/appoptics/appoptics-apm-node/blob/master/lib/loggers.js#L114)